packages/create-contentful-app/lib/create-app-definition.js

```diff
@@ -0,0 +1,91 @@
+/* eslint-disable no-console, no-process-exit */
+
+const inquirer = require('inquirer');
+const path = require('path');
+const { createClient } = require('contentful-management');
+const login = require('./login');
+
+async function fetchOrganizations(client) {
+  try {
+    const orgs = await client.getOrganizations();
+
+    return orgs.items.map(org => ({
+      name: `${org.name} (${org.sys.id})`,
+      value: org.sys.id
+    }));
+  } catch (err) {
+    console.log();
+    console.log('Could not fetch your organizations. Make sure you provided a valid access token.');
+    console.log();
+    console.log(err.message);
+    process.exit(1);
+  }
+}
+
+module.exports = async function createAppDefition(appDefinitionSettings = {}) {
+  let selectedOrg;
+
+  const accessToken = await login();
+
+  const client = createClient({
+    accessToken
+  });
+
+  const organizations = await fetchOrganizations(client);
+
+  const { organizationId } = await inquirer.prompt([
+    {
+      name: 'organizationId',
+      message: 'Select an organization for your app:',
+      type: 'list',
+      choices: organizations
+    }
+  ]);
+  selectedOrg = organizations.find(org => org.value === organizationId);
+
+  const appName = appDefinitionSettings.name || path.basename(process.cwd());
+  const body = {
+    name: appName,
+    src: 'http://localhost:3000',
```

Comment:
Does it match the default port of `npm start`? It was `1234` in CCE

Reply:
Yep - CRA defaults to 3000